### PR TITLE
Jenkinsfiles: Multiple fixes

### PR DESCRIPTION
**Pull request by eloycoto**
Multiple changes, the main idea is in the near future to build a Jenkins Library in groovy, that will help us to feed common task like (Failfast, fetch github tabs, destroy vms, etc.. ). and use a single line in our test. 

The changes are the following: 

- Nightly: Added Kubernetes 1.8 to the Nightly test.
- Ginkgo-All: Using kubernetes 1.9
- Ginkgo-kubernetes: Added failfast Option in the builds
- Ginkgo: Using failfast if no master branch
- Ginkgo-init: added 1.9 as the default server

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

**Comment by aanm** [comment]
We are testing k8s 1.10 and 1.11. We expect 1.11 to fail sometimes sense it's in alpha state. If 1.11 fails does it means 1.10 will not finish because of `failFast`?

**Comment by eloycoto** [comment]
yes

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-missed-k8s

**Comment by eloycoto** [comment]
test-missed-k8s

**Comment by eloycoto** [comment]
test-me-please

File: ginkgo.Jenkinsfile [hunk 1]
```diff
@@ -57,10 +67,10 @@ pipeline {
                         sh 'cd ${TESTDIR}; ginkgo --focus="RuntimeValidated*" -v -noColor'
                     },
                     "K8s-1.9":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.9 ginkgo --focus=" K8sValidated*" -v -noColor'
+                        sh 'cd ${TESTDIR}; K8S_VERSION=1.9 ginkgo --focus=" K8sValidated*" -v -noColor ${FAILFAST}'
```
Comment (ianvernon) [inline ginkgo.Jenkinsfile:68]: Have you validated whether the `AfterEach` blocks get executed if a test fails with `-failFast` enabled? I want to make sure we have logs to triage / examine / peruse if builds fail with this option enabled.

**Review by ianvernon — COMMENTED** [review]
Please check log-gathering functionality if `failFast` is enabled.

File: ginkgo.Jenkinsfile (hunk 1 above)
Comment (eloycoto) [reply ginkgo.Jenkinsfile:68]: Yep, `AfterEach` is called:

Example: 
```
package gink_example_test

import (
	"fmt"
	"testing"

	. "github.com/eloycoto/ginkgo-ext"
	"github.com/onsi/ginkgo/reporters"

	"github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
)

func TestGinkExample(t *testing.T) {
	RegisterFailHandler(Fail)
	junitReporter := reporters.NewJUnitReporter("junit.xml")
	RunSpecsWithDefaultAndCustomReporters(t, "Foo Suite", []ginkgo.Reporter{junitReporter})
}

var _ = Describe("K8Spolicies", func() {
	AfterEach(func() {
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("AfterEach")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
	})

	It("Test1", func() {
		Expect(true).To(BeFalse())
	})
})
```

```
----------------------                 
----------------------                 
----------------------                 
AfterEach                              
----------------------                 
----------------------                 
----------------------                 
• Failure [0.001 seconds]              
K8Spolicies                            
/home/ecoto/.go/src/github.com/eloycoto/ginkgo-ext/scopes.go:170               
  Test1 [It]                           
  /home/ecoto/.go/src/github.com/eloycoto/ginkgo-ext/scopes.go:202             

  Expected                             
      <bool>: true                     
  to be false                          

  /home/ecoto/.go/src/github.com/eloycoto/gink_example/gink_example_suite_test.go:32           
------------------------------         


Summarizing 1 Failure:                 

[Fail] K8Spolicies [It] Test1          
/home/ecoto/.go/src/github.com/eloycoto/gink_example/gink_example_suite_test.go:32             

Ran 1 of 1 Specs in 0.001 seconds      
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestGinkExample (0.00s)         
FAIL                                   
```

**Comment by ianvernon** [comment]
@eloycoto needs rebase.

**Review by ianvernon — APPROVED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Pull request merged**